cmake/FindSYSTEMD.cmake

```diff
@@ -110,16 +110,29 @@ pkg_check_modules(SYSTEMD_PROGRAM QUIET systemd)
 
 if (SYSTEMD_PROGRAM_FOUND)
     if ("${SYSTEMD_UNIT_DIR}" STREQUAL "")
-        # Use pkg-config to look up the systemd unit install directory
+        # Use pkg-config to look up the systemd system unit install directory
         execute_process(COMMAND ${PKG_CONFIG_EXECUTABLE}
             --variable=systemdsystemunitdir systemd
             OUTPUT_VARIABLE SYSTEMD_UNIT_DIR)
         string(REGEX REPLACE "[ \t\n]+" "" SYSTEMD_UNIT_DIR "${SYSTEMD_UNIT_DIR}")
     endif()
 
-    message(STATUS "systemd services install dir: ${SYSTEMD_UNIT_DIR}")
+    message(STATUS "systemd system services install dir: ${SYSTEMD_UNIT_DIR}")
+
+    if ("${SYSTEMD_USER_UNIT_DIR}" STREQUAL "")
+        # Use pkg-config to look up the systemd user unit install directory
+        execute_process(COMMAND ${PKG_CONFIG_EXECUTABLE}
+            --variable=systemduserunitdir systemd
+            OUTPUT_VARIABLE SYSTEMD_USER_UNIT_DIR)
```

Comment:
It may also be nice to have the main CMakeList.txt summary print out this variable next to the other one:
https://github.com/Cisco-Talos/clamav/blob/90a9fbd369dcf58d7d35c5fc7cabb6e972048ec7/CMakeLists.txt#L1256

Reply:
@nils-werner I think we forgot about this.